### PR TITLE
fix for nbd instructions

### PR DESCRIPTION
**Pull request by SpectralHiss**

### CODE REVIEW

File: INSTALL_STORE_CONTAINER.md [hunk 1]
```diff
@@ -86,10 +86,16 @@ $ sudo mkdir /var/lib/storageos
 $ sudo modprobe nbd nbds_max=1024
 ```
 
-or add the following line to `/etc/modules`:
+**In order for the NBD module to be loaded on reboot:**
```
Comment (oicheryl) [inline INSTALL_STORE_CONTAINER.md]: Use the wording "To ensure the NBD module is loaded on reboot:"

File: INSTALL_STORE_CONTAINER.md [hunk 2]
```diff
@@ -86,10 +86,16 @@ $ sudo mkdir /var/lib/storageos
...
 
+1. Add the following line to `/etc/modules`
 ```
-nbd nbds_max=1024
+nbd
+```
+
+1. Add the following module configuration line in `/etc/modprobe.d/nbd.conf`
```
Comment (oicheryl) [inline INSTALL_STORE_CONTAINER.md]: The numbering should be 2., here and in other instances.

Comment (SpectralHiss) [inline INSTALL_STORE_CONTAINER.md]: it translates to 2 in the markdown.. this is a common thing

File: INSTALL_STORE_CONTAINER.md (hunk 1 above)
Comment (SpectralHiss) [inline INSTALL_STORE_CONTAINER.md]: Ok coming up

File: INSTALL_STORE_CONTAINER.md (hunk 2 above)
Comment (oicheryl) [inline INSTALL_STORE_CONTAINER.md]: I'm aware, but unless there's a specific reason I'd rather the numbering is correct in the source.

**Pull request merged**